packages/cashier/src/pages/payment-agent/payment-agent-list/payment-agent-list.tsx

```diff
@@ -10,8 +9,32 @@ import PaymentAgentContainer from '../payment-agent-container';
 import PaymentAgentWithdrawalLocked from '../payment-agent-withdrawal-locked';
 import PaymentAgentDisclaimer from '../payment-agent-disclaimer';
 import SideNote from 'Components/side-note';
+import { TRootStore, TSideNotesProps, TServerError } from 'Types';
 import './payment-agent-list.scss';
 
+declare module 'react' {
+    interface HTMLAttributes<T> extends React.AriaAttributes, React.DOMAttributes<T> {
+        label?: string;
+    }
+}
```

Comment:
@hamid-deriv Let's remove this and have the error instead, It's better to refactor the `Tabs` component and fix there later instead of patching here and then forgetting about it.

Reply:
Yes, you're right. I will add TODO for it as this is used in other components, too. So we can check the TODO after refactoring the Tab component.